notebooks/rbc-omnisci-ibis.ipynb

```diff
@@ -2,39 +2,39 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
     "import ibis\n",
-    "from rbc.mapd import RemoteMapD"
+    "from rbc.omniscidb import RemoteOmnisci"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [],
    "source": [
-    "mapd = RemoteMapD()  # RemoteMapD(user=, password=, host=, port=, dbname=)\n",
-    "mapd_ibis = ibis.mapd.connect(user=mapd.user, password=mapd.password,\n",
-    "                              host=mapd.host, port=mapd.port, database=mapd.dbname)"
+    "omni = RemoteOmnisci()  # RemoteOmisci(user=, password=, host=, port=, dbname=)\n",
+    "omni_ibis = ibis.omniscidb.connect(user=omni.user, password=omni.password,\n",
+    "                              host=omni.host, port=omni.port, database=omni.dbname)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# ibis.mapd bug workaround:\n",
+    "# ibis.omniscidb bug workaround:\n",
     "import sys\n",
-    "sys.modules['ibis.mapd.operations']._sql_type_names['float32'] = 'float'"
+    "sys.modules['ibis.omniscidb.operations']._sql_type_names['float32'] = 'float'"
```

Comment:
`ibis` does not have `omniscidb`. Restore it as `mapd`.

Reply:
See my comment above.

Reply:
noop.